libs/components/pages/src/lib/modules/page/page-links.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, inject } from '@angular/core';
+import { SkyLayoutHostService } from '@skyux/core';
+
+/**
+ * Displays page links within a block page layout.
+ * @internal
+ */
+@Component({
+  standalone: true,
+  selector: 'sky-page-links',
+  template: '<ng-content/>',
+  styles: `
+    :host {
+      display: block;
+    }
+  `,
+})
+export class SkyPageLinksComponent {
+  constructor() {
+    inject(SkyLayoutHostService).setHostLayoutForChild({
```

Comment:
This service was meant to communicate a required page layout from a component in the content area to the outer page component, and there can be only one. My concern with using it here is that you may have a component in the content area request a layout that is then overridden by this page links component since there can only be one. I know there's not a use case for anything other than a blocks layout having page links, but I'd like to ensure it works with any layout should the need arise so we don't paint ourselves into a corner.

Since page and page links are tightly coupled, can we make a purpose-built service provided by the page component and injected by the page links component? We can also make this injectable required which will prevent consumers from using the component outside of a page component.

Reply:
I switched to a sibling CSS selector, which removed the need for an additional service and CSS class.